src/utils/validators.js

```diff
@@ -5,6 +5,7 @@ const PERMALINK_REGEX = '^(/([a-z]+([-][a-z]+)*/)+)$';
 const permalinkRegexTest = RegExp(PERMALINK_REGEX);
 const DATE_REGEX = '^([0-9]{4}-[0-9]{2}-[0-9]{2})$';
 const dateRegexTest = RegExp(DATE_REGEX);
+const fileNameRegexTest = /^[\w,\s-]+\.[A-Za-z]{3}$/;
```

Comment:
Perhaps we need to discuss what kind of file name to accept since there are some file names which Github accepts that fail this test such as `Screen Shot 2019-12-10 at 4.40.02 PM.png`

Another thing to consider is if we will run all file names through the same regex tests. If not, we might want to consider renaming this variable